runtime/near-vm-runner/src/tests/cache.rs

```diff
@@ -63,8 +63,8 @@ fn test_does_not_cache_io_error() {
     let config = test_vm_config();
     with_vm_variants(&config, |vm_kind: VMKind| {
         match vm_kind {
-            VMKind::Wasmer0 | VMKind::Wasmer2 | VMKind::NearVm => {}
-            VMKind::Wasmtime => return,
+            VMKind::NearVm => {}
+            VMKind::Wasmer0 | VMKind::Wasmer2 | VMKind::Wasmtime => return,
```

Comment:
cc @Ekleog-NEAR this started failing after my changes because the old wasmer implementations do not interact with cache and the previous `Some(VMRunnerError::CacheError(crate::logic::errors::CacheError::ReadError(_)))` was matching only by accident of missing contract error repurposing this error variant until now.